frontend/www/js/omegaup/components/problem/Details.vue

```diff
@@ -374,7 +347,8 @@ export default class ProblemDetails extends Vue {
   @Prop() selectedPrivateTags!: string[];
   @Prop() hasBeenNominated!: boolean;
   @Prop({ default: null }) runDetailsData!: types.RunDetails | null;
-  @Prop() guid!: string;
+  @Prop({ default: null }) guid!: string;
+  @Prop({ default: null }) problemAlias!: string;
```

Comment:
```suggestion
  @Prop({ default: null }) guid!: string | null;
  @Prop({ default: null }) problemAlias!: string | null;
```

recuerda que es ilegal ponerle `null` de default a algo que no se declaró que puede legalmente contener `null`.

Reply:
Listo.